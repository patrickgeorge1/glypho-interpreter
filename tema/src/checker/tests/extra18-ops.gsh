ii*d1+<!o
