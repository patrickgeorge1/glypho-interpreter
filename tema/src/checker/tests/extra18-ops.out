9999999999999999899999999999999999900000000000000002
